modules/packer/custom-image/README.md

```diff
@@ -30,6 +30,29 @@ images to internal projects.
 [hpcimage]: https://cloud.google.com/compute/docs/instances/create-hpc-vm
 [startup-metadata]: https://cloud.google.com/compute/docs/instances/startup-scripts/linux
 
+## Example blueprints
+
+Several examples blueprints that create a startup-scripts using the HPC Toolkit [startup-script] module and then use it to build a specialized image can be found below.
+
+### [Image Builder]
+The [Image Builder] blueprint demonstrates a solution that builds an image using:
+
+* The [HPC VM Image][hpcimage] as a base upon which to customize
+* A VPC network with firewall rules that allow IAP-based SSH tunnels
+* A Toolkit runner that installs a custom script
+
+Please review the [examples README] for usage instructions.
+
+### Intel-Optimized Slurm Cluster
+
+The [Intel-Optimized] Slurm Cluster [blueprint](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/community/examples/intel/hpc-cluster-intel-select.yaml)
+Intel Select Solutions within the Toolkit. It adds the intel compliance software on top of a Slurm on GCP image.
+
+[Image Builder]: https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/examples/image-builder.yaml
+[startup-script]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/modules/scripts/startup-script
+[examples README]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/examples#image-builderyaml-
+[Intel-Optimized]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/examples/intel#intel-optimized-slurm-cluster
```

Comment:
It is recommended to use relative paths for all of these as it will preserve the branch being viewed. All these links point to main, but if you use relative path then if I am browsing from the develop branch the link will keep me on the develop branch.

Reply:
oh, I did not know that, thanks!

Reply:
changed